maestro-client/src/main/java/maestro/drivers/AndroidDriver.kt

```diff
@@ -180,6 +181,8 @@ class AndroidDriver(
             val apkFile = AndroidAppFiles.getApkFile(dadb, appId)
             val manifest = apkFile.asManifest()
             runCatching {
+                val sessionUUID = sessionId ?: UUID.randomUUID()
+                dadb.shell("setprop debug.maestro.sessionId $sessionUUID")
```

Comment:
Do network mocking and all related features currently support emulator or can be physical device as well?

Reply:
I'm not sure if physical devices will work as well. Didn't tested on them